index1.html

```diff
@@ -0,0 +1,236 @@
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <button class="btn">See Project</button>
+            </div>
+        </div>
+
+        <div class="project">
+            <div class="work-img">
+                <img src="/images1/multi-post.svg" alt="multi-post image">
+            </div>
+            <div class="work-description">
+                <h2 class="work-title">Multi-Post
+                    Stories</h2>
+                <ul class="canopy">
+                    <li>canopy</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p class="work-details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="work-tools">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <button class="btn">See Project</button>
+            </div>
+        </div>
+       
+        <div class="project">
+            <div class="work-img">
+                <img src="/images1/3.svg" alt="tonic3 image">
+            </div>
+            <div class="work-description">
+                <h2 class="work-title">tonic</h2>
+                <ul class="canopy">
+                    <li>canopy</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p class="work-details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="work-tools">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <button class="btn">See Project</button>
+            </div>
+        </div>
+        <div class="project">
+            <div class="work-img">
+                <img src="/images1/4.svg" alt="tonic image">
+            </div>
+            <div class="work-description">
+                <h2 class="work-title">Multi-Post Stories</h2>
+                <ul class="canopy">
+                    <li>canopy</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p class="work-details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="work-tools">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <button class="btn">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section id="about">
+        <div class="about-container">
+            <p class="myself">About Myself</p>
```

Comment:
- [ ] Please kindly use any of the header tags aside h1, as suggested earlier. 👍 
```suggestion
            <h2 class="myself">About Myself</h2>
```